File: examples-sdk/java/303-rag-hybrid-exact-semantic/Main.java
// 303-rag-hybrid-exact-semantic — Java SDK equivalent
// Compile: vil compile --from java --input 303-rag-hybrid-exact-semantic/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("rag-hybrid-exact-semantic", 3112);
        ServiceProcess rag_hybrid = new ServiceProcess("rag-hybrid");
        rag_hybrid.endpoint("POST", "/hybrid", "hybrid_handler");
        server.service(rag_hybrid);
        server.compile();
    }
}

File: examples-sdk/swift/303-rag-hybrid-exact-semantic/main.swift
// 303-rag-hybrid-exact-semantic — Swift SDK equivalent
// Compile: vil compile --from swift --input 303-rag-hybrid-exact-semantic/main.swift --release

let server = VilServer(name: "rag-hybrid-exact-semantic", port: 3112)
let rag_hybrid = ServiceProcess(name: "rag-hybrid")
rag_hybrid.endpoint(method: "POST", path: "/hybrid", handler: "hybrid_handler")
server.service(rag_hybrid)
server.compile()
